### PR TITLE
HOTFIX: re-wire style for jquery UI checkbox radio buttons.

### PR DESCRIPTION
**Pull request by nstriedinger**
## Ticket(s)
**Do not** submit a Pull Request without a relevant JIRA ticket or Github issue! If you are creating a new pattern or feature, create an issue describing the need for this feature in Github **first** and then link to it below.

**Github Issue**
- [Issue XXX: Issue Title](https://github.com/AmericanMedicalAssociation/AMA-style-guide/issues/XXX)

**Jira Ticket**
- [EWL-XXXX: Ticket Title](https://issues.ama-assn.org/browse/EWL-XXXX)

## Description
Explain the technical implementation of the work done.

## To Test
- [ ] Add detailed steps for reviewers to test this feature.
- [ ] Did you test in IE 11?

## Visual Regressions

_Please provide the pull request ID below. This will then reference the automated VRT report after it has been generated._

A report is available [here](http://htmlpreview.github.io/?https://github.com/AmericanMedicalAssociation/ama-style-guide-2/blob/visual-regression-testing-artifact/[PULL_REQUEST_ID]/html_report/index.html).

**Before proceeding:** Run visual regression tests locally to ensure new work does not introduce new bugs.

_The visual regression tests always compare against the production version of the style guide. Changes and new work will show up as test failures. Please describe your work so that the report can be reviewed by the team._

If you are creating or updating a pattern be sure you have created or updated the [scenario in `backstop.json`](ama-style-guide-2/styleguide/backstop.json).

_After changes in the report are documented and demonstrate your desired changes, please mark the pull request as `ready for review`.

For more information on visual regression testing review the [How do I run tests?](https://issues.ama-assn.org:8446/confluence/pages/viewpage.action?pageId=23298568) document in Confluence.

## Relevant Screenshots/GIFs
Use something like [Skitch](https://evernote.com/skitch/) or [GIPHY Capture](https://giphy.com/apps/giphycapture) to capture images/gifs to demonstrate behaviors.

## Remaining Tasks
Things relevant to here may be updating [The Glossary](https://issues.ama-assn.org:8446/confluence/display/DTD/Glossary?src=contextnavpagetreemode) with the latest naming changes or review of the relative [D8](https://github.com/AmericanMedicalAssociation/ama-d8) work.

## Additional Notes
Anything more to add? Good things to call out here are:
- are there any PRs in this or other repositories that relate to or affect this PR?
- are there any caveats or oddities testers should know about when testing this PR?
- are there any problems you ran into during your work that you'd like to call out?

---

[Guidelines for Contribution](CONTRIBUTING.md)
[SG2 Standards](ama-style-guide-2/docs/standards.md)

### CODE REVIEW

File: styleguide/source/assets/js/webforms.js [hunk 1]
```diff
@@ -72,6 +72,12 @@
           verifyFields($contactForm);
         }
 
+        $inputs.on('click', function () {
+          console.log($(this).closest('.form-item').siblings().find(this));
```
Comment (designerneil) [inline styleguide/source/assets/js/webforms.js]: can we remove this?

File: styleguide/source/assets/js/webforms.js [hunk 2]
```diff
@@ -72,6 +72,12 @@
...
+          $(this).addClass('active').next().addClass('active');
+          $(this).closest('.ama__radio').siblings().find(':input').removeClass('active').next().removeClass('active');
+        });
+
         $inputs.on('focus change keypress selectmenuchange', function () {
           var iconClass = 'edit';
           $closestSection = $(this).closest('section');
```
Comment (designerneil) [inline styleguide/source/assets/js/webforms.js:82]: This seems error prone. Can we use names instead of closest?

**Comment by labbydev** [comment]
When using this locally I'm still not able to get the form to load correctly without redirecting when I'm logged in. When I'm logged out it works, but select lists are not styled correctly. And it seems like errors are being thrown way to easily but that may just be me. 

ama-one local_form_contact-us 1

**Comment by JaeBea** [comment]
@nstriedinger ,
Please resolve merge conflict and update PR to include a description explaining the purpose of your work.

**Comment by designerneil** [comment]
@nstriedinger is this PR still needed? It seems like most of the issues in this PR have been resolved and merged into developed

**Pull request closed without merging**